app/controllers/clearance/sessions_controller.rb

```diff
@@ -33,4 +40,8 @@ def url_after_create
   def url_after_destroy
     sign_in_url
   end
+
+  def url_for_signed_in_users
+    url_after_create
```

Comment:
I'm hesitant to add a configuration option dedicated for this purpose. It's already factored out into a method which can be overridden if desired and is ultimately backed by `Clearance.configuration.redirect_url`.

Is there a use case where you would expect these two URLs to be different?